platform/tracing/opentracing.go

```diff
@@ -1,10 +1,15 @@
 package tracing
 
 import (
+	"context"
+
 	"contrib.go.opencensus.io/exporter/jaeger"
 	"go.opencensus.io/trace"
 )
 
+// Span represents an individual unit of work in the system
+type Span = trace.Span
```

Comment:
Is it the intended syntax? Go as two kind of `type` declaration one is `type MyType = OtherType` to alias a type and the other one is `type MyType OtherType` to wrap a type.

The first one has mainly been introduced to help break big refactoring in smaller steps:
1. we create an alias type `type MyType OtherType`
2. we change all the places where we pass `OtherType` as parameter to use `MyType` as they can be used interchangeably
3. we add the new API functions we need on `OtherType`
4. we replace the old functions calls to use the new API
5. we remove the `=` in the type declaration to make sure the functions of `MyType` are no longer accessible

It's a good pattern but I'm not sure we are in such case here. 

I would suggest using `type MyType OtherType` and directly gets a wrapper. I'm not sure we really need this refactoring steps as we don't really pass `trace.Span` structs around nor use a ton of function on them (except `End`).

Reply:
I agree with you kevin, should we then add the `(span Span) End()` function ?

Reply:
It was actually intended but I did not think the refactoring would really be an issue as it would all just be hidden behind the custom type and changing the original type would thus not be an issue.

But will make the edits

Reply:
> It was actually intended but I did not think the refactoring would really be an issue as it would all just be hidden behind the custom type

It won't be an issue but my issue with the current implementation it that we don't hide anything behind a custom type, we just create another name for the exact same thing.